index.html

```diff
@@ -36,7 +36,7 @@
           class="btn-login"
           href='https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97'
         >
-          Login
+        <button class="button">Sign In With Github <img class="icon" src="img/github.png" alt="Github Icon" /></button>
```

Comment:
Wouldn't something like `button class="login-btn-txt"` make more sense?
The class name should be self explanatory about it's functioning, right?

Same for img as well. `img class="logo"` would be more self explanatory.

Reply:
Ok. Let me change that

Reply:
Have done the changes. Please check them.